.sonarcloud.properties

```diff
@@ -1,12 +1,13 @@
+## Edit .github/workflows/run_sonarcloud_scan.yaml instead
```

Comment:
If these are all commented, do we even need this file? I know @jeremy-page was working on this. FE scans are working off of the `run_sonarcloud_scan` GH action, but I think BE scans were maybe still using this file?

Reply:
I _tried_ modifying `.sonarcloud.properties` and it didn't seem to do anything. I was taking the more conservative approach of leaving it, but maybe we should just remove it.

Reply:
The original was good at reducing duplications and related code smells.

Moving the list from exclusions to tests may have the desired effect (if uncommented).

Near as I can tell, .sonarcloud.properties testing in a branch is ineffective.